.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
```

Comment:
```suggestion
  test:
```

Or did you mean this?

Reply:
see https://github.com/standard/standard/pull/1629#discussion_r566862061